index.js

```diff
@@ -89,7 +94,7 @@ Filter.prototype.processAndCacheFile = function (srcDir, destDir, relativePath)
         throw err
       })
       .then(function (cacheInfo) {
-        copyToCache(cacheInfo)
+        return copyToCache(cacheInfo)
```

Comment:
Ditto here, that `return` isn't needed, is it?

I omitted both of those from my merge at #22. If those changes are actually necessary, do let me know please.